README.md

```diff
@@ -34,3 +34,10 @@ Class crates
 
 * [usbd-serial](https://github.com/mvirkkunen/usbd-serial) [![Crates.io](https://img.shields.io/crates/v/usbd-serial.svg)](https://crates.io/crates/usbd-serial) - CDC-ACM serial port class
 * [usbd-hid](https://github.com/twitchyliquid64/usbd-hid) [![Crates.io](https://img.shields.io/crates/v/usbd-hid.svg)](https://crates.io/crates/usbd-hid) - HID class
+
+
+Others
+------
+
+Other implementations for USB in Rust
+* [embassy-usb](https://github.com/embassy-rs/embassy/blob/master/embassy-usb/src/driver.rs)
```

Comment:
It would be good to explicitly note that embassy-usb is an async variant. Async isn't widely available on stable Rust at the moment due to GATs.